default.nix

```diff
@@ -285,28 +283,20 @@ let overrideCabal = pkg: f: if pkg == null then null else haskellLib.overrideCab
         ########################################################################
         # Tweaks
         ########################################################################
-        gi-atk = dontUseOverloads super.gi-atk;
-        gi-cairo = dontUseOverloads super.gi-cairo;
-        gi-gdk = dontUseOverloads super.gi-gdk;
-        gi-gdkpixbuf = dontUseOverloads super.gi-gdkpixbuf;
-        gi-glib = dontUseOverloads (self.callPackage ./gi-glib.nix {});
-        gi-gio = dontUseOverloads (self.callPackage ./gi-gio.nix {});
-        gi-gtk = dontUseOverloads (self.callPackage ./gi-gtk.nix {
+        gi-glib = self.callPackage ./gi-glib.nix {};
+        gi-gio = self.callPackage ./gi-gio.nix {};
+        gi-gtk = self.callPackage ./gi-gtk.nix {
           gtk3 = nixpkgs.gnome3.gtk;
-        });
-        gi-javascriptcore = dontUseOverloads (self.callPackage ./gi-javascriptcore.nix {});
-        gi-webkit2 = dontUseOverloads (self.callPackage ./gi-webkit2.nix {
+        };
+        gi-javascriptcore = self.callPackage ./gi-javascriptcore.nix {};
+        gi-webkit2 = self.callPackage ./gi-webkit2.nix {
           webkitgtk = nixpkgs.webkitgtk216x;
-        });
-        gi-gobject = dontUseOverloads super.gi-gobject;
-        gi-pango = dontUseOverloads super.gi-pango;
-        gi-soup = dontUseOverloads super.gi-soup;
-        gi-webkit = dontUseOverloads super.gi-webkit;
-        gi-gtksource = dontUseOverloads (super.gi-gtksource.override {
+        };
+        gi-gtksource = super.gi-gtksource.override {
           inherit (nixpkgs.gnome3) gtksourceview;
-        });
+        };
 
-        haskell-gi-overloading = self.callPackage ./haskell-gi-overloading.nix {};
```

Comment:
Instead of this, I think we could just do `self.callHackage "haskell-gi-overloading" "0.0" {}`.  That way we shouldn't need the external file.

Reply:
I think you might be looking at the deleted line instead of the added one.  The added line uses `haskell-gi-overloading_0_0` from nixpkgs:

    haskell-gi-overloading = super.haskell-gi-overloading_0_0;

Reply:
0_0

Reply:
Whoops, my bad!  Thanks for clarifying.